ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json

```diff
@@ -358,5 +358,10 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "grantAwaitingDocumentatioNotificationSent", "Label": "Has a grant awaiting documentation notification been sent?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "pcqId", "Label": "PCQ ID", "FieldType": "Text", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "reprintDocument", "Label": "Which document do you wish to reprint?",  "FieldType": "DynamicList", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "reprintNumberOfCopies", "Label": "How many copies are needed?", "FieldType": "Number", "SecurityClassification": "Public"}
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "reprintNumberOfCopies", "Label": "How many copies are needed?", "FieldType": "Number", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "nameOfFirmNamedInWill", "Label": "Name of firm named in will", "FieldType": "Text", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "nameOfExecutorApplying", "Label": "Name of executor applying", "FieldType": "Text", "SecurityClassification": "Public"},
```

Comment:
Could we have a more specific field name than nameOfExecutorApplying? I think there are going to be multiple points we add an executor name so it could get confusing

Reply:
ok good shout!